### ISSUE TITLE
Demucs fp32 got fail_accuracy

### ISSUE THREAD

**Issue by mengfei25** [intel/torch-xpu-ops#504]
### 🐛 Describe the bug

torchbench_float32_training
xpu  train demucs                             
E0626 07:50:03.811000 140442038515520 torch/_dynamo/utils.py:1478] RMSE (res-fp64): 0.03316, (ref-fp64): 0.00065 and shape=torch.Size([]). res.dtype: torch.float32, multiplier: 3.000000, tol: 0.001000
fail_accuracy

loading model: 0it [00:00, ?it/s]
loading model: 0it [00:26, ?it/s]

### Versions

torch-xpu-ops: https://github.com/intel/torch-xpu-ops/commit/31c400195d63064940242220dc9100322d36bac4
pytorch: 0f81473d7b4a1bf09246410712df22541be7caf3 + PRs: 127277,129120
device: PVC 1100, 803.61, 0.5.1

**Comment by weishi-deng** [comment]
Known issue. Use to be closed because it's a common issue for all backends: CPU, CUDA, XPU.

**Comment by retonym** [comment]
similar issue in https://github.com/intel/torch-xpu-ops/issues/488.
will close this issue, and only track issues their with adding fp32 label
